.github/workflows/docs.yml

```diff
@@ -6,7 +6,7 @@ on:
       - 'v*'
 
 jobs:
-  Documentation:
+  build:
```

Comment:
Why change the name of this action? `Documentation` seemed more appropriate...

Reply:
Sure, I can revert that change, I just followed the recommended names from the official doc [page](https://github.com/actions/deploy-pages)

Reply:
I see. You are probably right then... but, maybe then let's refactor all the workflows appropriately in the future (once we successfully release). thx!